proforma/proforma_generator.py

```diff
@@ -518,93 +518,44 @@ def fill_alignment(ws, series, current_row, alignment):
                            number_format='##%')
     current_row += 1
     ws['A{}'.format(current_row)] = "Year one total site carbon dioxide emissions (lb CO2 equivalent)"
-    ws['B{}'.format(current_row)] = site.year_one_emissions_lb_C02
+    ws['B{}'.format(current_row)] = site.year_one_emissions_tCO2
     make_attribute_row(ws, current_row)
     current_row += 1
 
     ws['A{}'.format(current_row)] = "Year one total site carbon dioxide emissions BAU (lb CO2 equivalent)"
-    ws['B{}'.format(current_row)] = site.year_one_emissions_bau_lb_C02
+    ws['B{}'.format(current_row)] = site.year_one_emissions_tCO2_bau
     make_attribute_row(ws, current_row)
     current_row += 1
     
     ws['A{}'.format(current_row)] = "Year one total carbon dioxide emissions from electric utility purchases (lb CO2 equivalent)"
-    ws['B{}'.format(current_row)] = electric_tariff.year_one_emissions_lb_C02
+    ws['B{}'.format(current_row)] = electric_tariff.year_one_emissions_tCO2
     make_attribute_row(ws, current_row)
     current_row += 1
     
     ws['A{}'.format(current_row)] = "Year one total carbon dioxide emissions from electric utility purchases BAU (lb CO2 equivalent)"
-    ws['B{}'.format(current_row)] = electric_tariff.year_one_emissions_bau_lb_C02
-    make_attribute_row(ws, current_row)
-    current_row += 1
-    
-    ws['A{}'.format(current_row)] = "Year one total carbon dioxide emissions from backup generator use (lb CO2 equivalent)"
-    ws['B{}'.format(current_row)] = generator.year_one_emissions_lb_C02
-    make_attribute_row(ws, current_row)
-    current_row += 1
-    
-    ws['A{}'.format(current_row)] = "Year one total carbon dioxide emissions from backup generator use BAU (lb CO2 equivalent)"
-    ws['B{}'.format(current_row)] = generator.year_one_emissions_bau_lb_C02
-    make_attribute_row(ws, current_row)
-    current_row += 1
-
-    ws['A{}'.format(current_row)] = "Year one total carbon dioxide emissions from boiler fuels (lb CO2 equivalent)"
-    ws['B{}'.format(current_row)] = boiler.year_one_emissions_lb_C02
-    make_attribute_row(ws, current_row)
-    current_row += 1
-
-    ws['A{}'.format(current_row)] = "Year one total carbon dioxide emissions from boiler fuels BAU (lb CO2 equivalent)"
-    ws['B{}'.format(current_row)] = boiler.year_one_emissions_bau_lb_C02
-    make_attribute_row(ws, current_row)
-    current_row += 1
-
-    ws['A{}'.format(current_row)] = "Year one total carbon dioxide emissions from CHP fuels (lb CO2 equivalent)"
-    ws['B{}'.format(current_row)] = chp.year_one_emissions_lb_C02
+    ws['B{}'.format(current_row)] = electric_tariff.year_one_emissions_tCO2_bau
     make_attribute_row(ws, current_row)
     current_row += 1
 
     ws['A{}'.format(current_row)] = "Year one total site NOx emissions (lb NOx equivalent)"
```

Comment:
@Bill-Becker do you know why we only report CO2 and NOx emissions and not SO2 and PM2.5? Suggest either removing NOx or adding SO2 and PM2.5 unless there is a reason they were not included to being with.